exercises/house/example.js

```diff
@@ -0,0 +1,63 @@
+class House {
+  constructor() {
+    this.objects = [
```

Comment:
May I suggest moving `this.objects` and `this.actions` outside of the constructor?  They are really more like static properties of the class, rather than properties of each instance.  No reason to create new ones for every object, just as a best practice.    

Also, they are not part of the public API of this class, so if you leave them outside the class, but inside the module, they are not exposed to consumers of the class.

So...

```javascript

const OBJECTS = [
...
];

const ACTIONS = [
...
];

class House {
...
}
```

Then you would later reference these using `OBJECTS` and `ACTIONS` rather than `this.objects` and `this.actions`.

Reply:
Sounds much more sensible, thanks!